pages/01.HDM Documentation/06.vcd/40.Deployment Steps/default.md

```diff
@@ -391,6 +393,13 @@ _Figure 27: Network Mapping_
 
 10. To submit the request and initiate the cloud deployment, select _FINISH_. This will create deployment tasks for the cloud components, which can be monitored in vCenter Tasks. The progress can also be seen in the HDM Clouds view (figure 29).
 
+>>> **Attention**: 
+>>> After you submit the request the cloud deployment can take from 45 minutes to 
+>>> over an hour depending on storage and network performance. Please note
+>>> that the progress bar desplayed during this process is not linear and 
+>>> at 1% it takes about 25 minutes and at 50% another
```

Comment:
At 1% it takes 40-50 Mins for HDM on-cloud deployment task, as we upload the required packages to Organization VDC. Under Organization VDC upload ISO stays at 50% for about 25 mins and then upload OVF stays at 1% for about 20 mins.

Reply:
Rather elaborate and complicated to spell out the times and percentages, but have made the change never the less.

fixed.